kubectl-plugin/src/resources/pool.rs

```diff
@@ -67,7 +65,7 @@ impl List for Pools {
         match RestClient::client().pools_api().get_pools().await {
             Ok(pools) => {
                 // Print table, json or yaml based on output format.
-                utils::print_table::<openapi::models::Pool>(output, pools);
+                utils::print_table::<Vec<openapi::models::Pool>>(output, pools);
```

Comment:
you probably don't need to specify the type you can just do `print_table(output, pools)` and the same for all the other calls of print_table.